tests/framework/installer/ceph_installer.go

```diff
@@ -449,6 +466,16 @@ func (h *CephInstaller) InstallRook(namespace, storeType string, usePVC bool, st
 		return false, err
 	}
 	logger.Infof("installed rook operator and cluster : %s on k8s %s", namespace, h.k8sVersion)
+
+	if rookVersion != Version1_2 {
+		if !h.k8shelper.IsPodInExpectedState("rook-ceph-admission-controller", onamespace, "Running") {
+			h.k8shelper.GetStatus(onamespace)
+			logger.Error("admission controller is not running")
+			return false, err
```

Comment:
Instead of returning false, I'd suggest and `assert.Fail()` and then allow the test to continue. This will raise an error to fail the test, but it won't prevent the test from running.